resource/ui/ts/models/timeseries.ts

```diff
@@ -60,12 +60,15 @@ module Models {
         /**
          * Query dispatches a single time series query to the server.
          */
-        function query(start:Date, end:Date, series:string[]):promise<QueryResultSet> {
+        function query(start:Date, end:Date, agg:QueryAggregator, series:string[]):promise<QueryResultSet> {
```

Comment:
This is kind of gross to keep adding things here; I think we need to move to a builder model, which works well in javascript.  Like:

```
var query = Query("cr.store.livebytes.1")
   .aggregate(QueryAggregator.AVG_RATE)
   .recent(10 * 6000 /* ten minutes */);

resultPromise = query();
```

However, that will have to wait until next week.